usr/sbin/registercloudguest

```diff
@@ -324,6 +324,7 @@ def setup_ltss_registration(
     Run SUSEConnect to register LTSS for this instance
     """
     logging.info('Running LTSS registration...')
```

Comment:
we could use `message = 'Running LTSS registration...this takes a little longer'` in the log and print